addons/sale_timesheet/models/project.py

```diff
@@ -305,12 +305,12 @@ def get_panel_data(self):
             'profitability_items': self._get_profitability_items(),
         }
 
-    def _get_sale_order_lines(self):
+    def _get_sale_order_lines_not_down_payments(self):
```

Comment:
In stable, we cannot change the signature of the method, that is we cannot change the return type, the parameters and the method name.
So, we should keep the method name as before. 😇 
```suggestion
    def _get_sale_order_lines(self):
```